apps/charterafrica/payload.config.js

```diff
@@ -13,6 +13,7 @@ const appURL = process.env.PAYLOAD_PUBLIC_APP_URL;
 
 const adapter = s3Adapter({
   config: {
+    region: process?.env?.S3_REGION,
```

Comment:
Isn't eslint complaining about this env var? Should be first added in the `turbo.json` here: https://github.com/CodeForAfrica/ui/blob/abc7ce406fe3a9cc7ba48b8882dfec4d61002ef5/turbo.json#L28-L30

Reply:
It is added already here: ["S3_REGION",](https://github.com/CodeForAfrica/ui/blob/3f30dd84aba7468282fe680255ab600067625db7/turbo.json#L31)